src/Discovery/src/Eureka/Configuration/EurekaClientOptions.cs

```diff
@@ -10,91 +10,80 @@ namespace Steeltoe.Discovery.Eureka.Configuration;
 
     internal TimeSpan RegistryFetchInterval => TimeSpan.FromSeconds(RegistryFetchIntervalSeconds);
 
     /// <summary>
-    /// Gets or sets the URI to use to obtain an OAuth2 access token. Configuration property: eureka:client:accessTokenUri.
+    /// Gets or sets the URL to obtain OAuth2 access token from, before connecting to the Eureka server.
     /// </summary>
     public string? AccessTokenUri { get; set; }
 
     /// <summary>
-    /// Gets or sets the secret to use to obtain an OAuth2 access token. Configuration property: eureka:client:clientSecret.
+    /// Gets or sets the secret for obtaining access token.
     /// </summary>
     public string? ClientSecret { get; set; }
 
     /// <summary>
-    /// Gets or sets the client ID to use to obtain an OAuth2 access token. Configuration property: eureka:client:clientId.
+    /// Gets or sets the client ID for obtaining access token.
     /// </summary>
     public string? ClientId { get; set; }
 
     /// <summary>
-    /// Gets or sets how often (in seconds) to fetch the registry information from the Eureka server. Configuration property:
-    /// eureka:client:registryFetchIntervalSeconds.
+    /// Gets or sets how often (in seconds) to fetch registry information from the Eureka server. Default value: 30.
     /// </summary>
-    public int RegistryFetchIntervalSeconds { get; set; } = DefaultRegistryFetchIntervalSeconds;
+    public int RegistryFetchIntervalSeconds { get; set; } = 30;
 
     /// <summary>
-    /// Gets or sets a value indicating whether this instance should register its information with Eureka server for discovery by others. In some cases, you
-    /// do not want your instances to be discovered whereas you just want to discover other instances. Configuration property:
-    /// eureka:client:shouldRegisterWithEureka.
+    /// Gets or sets a value indicating whether to register the running app as a service instance. Default value: true.
     /// </summary>
     public bool ShouldRegisterWithEureka { get; set; } = true;
 
     /// <summary>
-    /// Gets or sets a value indicating whether the Eureka client should disable fetching of deltas and should rather resort to getting the full registry
-    /// information. Note that the delta fetches can reduce the traffic tremendously, because the rate of change with the Eureka server is normally much
-    /// lower than the rate of fetches. The changes are effective at runtime at the next registry fetch cycle as specified by
-    /// <see cref="RegistryFetchIntervalSeconds" />. Configuration property: eureka:client:shouldDisableDelta.
+    /// Gets or sets a value indicating whether to fetch the full registry each time or fetch only deltas. Note that the delta fetches can reduce the traffic
+    /// tremendously, because the rate of change in the Eureka server is normally much lower than the rate of fetches. Default value: false.
     /// </summary>
     [ConfigurationKeyName("ShouldDisableDelta")]
```

Comment:
These entries that use `ConfigurationKeyName` seem like they might have a much stronger case for keeping the configuration property in the XML comments somewhere. Would it make sense to add them as <remarks>?

Reply:
We could, but we haven't done that anywhere else in Steeltoe, as far as I'm aware of. These comments will ultimately surface in the schema for autocompletion in `appsettings.json`, where the adapted key name is already known (our schema generation tool should take `[ConfigurationKeyName]` into account, if it doesn't do so already). When displayed in IntelliSense, the developer is already configuring from code and not `appsettings.json`, so it's not too helpful there. 

Based on that, do you still want them to stay?

Reply:
And I suppose this information would only show up when you're configuring the values in code, where you'd _probably_ not care as much about the specific configuration key name